service/api.go

```diff
@@ -184,6 +184,11 @@ func (s *gnomobileService) DeleteAccount(ctx context.Context, req *connect.Reque
 			return nil, err
 		}
 	}
+	if s.activeAccount != nil &&
+		(s.activeAccount.GetName() == req.Msg.NameOrBech32 || crypto.AddressToBech32(s.activeAccount.GetAddress()) == req.Msg.NameOrBech32) {
+		// The deleted account was the active account.
+		s.activeAccount = nil
```

Comment:
```suggestion
        s.lock.Lock()
		s.activeAccount = nil
		s.lock.Unlock()
```

Reply:
Yes, thank you. I did a force push to fix it.